docs/api-reference/io/xviz-format.md

```diff
@@ -0,0 +1,13 @@
+# XVIZFormat
+
+Describes the format of the XVIZ data.
+
+This values is returned from [XVIZData](/docs/api-reference/io/xviz-data.md) and used by the
+[XVIZFormatWriter](/docs/api-reference/io/xviz-format-writer.md)
+
+## Enumeration values
+
+- `BINARY` - GLB-based binary formatted data
```

Comment:
Do we want to use `GLB_BINARY` to give us more naming space?

Reply:
I thought about this but wasn't sure, as we could always do `BINARY_PROTOBUF`, but yes I aggress we can be more explicit here.

Reply:
Changed to `BINARY_GLB`